io/net/Network_config.py.data/Network_config.yaml

```diff
@@ -1,2 +1,4 @@
 Parameters:
     interface: "enP3p1s0f0"
+    host_ip: ""
+    netmask: ""
```

Comment:
Please take care of Readme file changes also, if necessary.

Reply:
ok